includes/error_status.h

```diff
@@ -22,6 +22,7 @@ class ErrorStatus : public ResponseStatus {
 	private:
 		ErrorStatus(void);
 		ErrorStatus(int, const std::string&);
+        ~ErrorStatus();
```

Comment:
僕のvscodeのタブ補完おかしそうですね。修正します。
```suggestion
		~ErrorStatus();
```